src/lib/_utilities/i18n.utilities.ts

```diff
@@ -2,7 +2,7 @@ import sanitize from 'xss';
 import { get } from 'svelte/store';
 import { z } from 'zod';
 
-import { strings } from '$lib/locale.store';
+import { stringsStore } from '$lib/locale.store';
```

Comment:
This is to help be more explicit throughout the code. I've had to adjust what's exported by most of these "stores", and changing the name to make it more quickly identifiable.